low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -325,21 +324,36 @@ def sspl_bundle_cleanup():
     if os.path.exists(sspl_bundle_tmp):
         shutil.rmtree(sspl_bundle_tmp)
 
+def validate_sb_request():
+    """Checking previous process pid/request is running or not."""
+    allow_new_request = True
+    sb_pid = None
+    if os.path.isfile(onGoingRequestPath):
+        with open(onGoingRequestPath, 'r') as f:
+            sb_pid = f.read()
+        if sb_pid and psutil.pid_exists(int(sb_pid)):
+            allow_new_request = False
+        else:
+            sspl_bundle_cleanup()
+    return allow_new_request
+
 if __name__ == "__main__":
-    if not os.path.isfile(onGoingRequestPath):
-        if len(sys.argv) < 3:
-            print("Unrecognized arguments: %s" % sys.argv)
-            sys.exit(1)
-        tmpFile = open(onGoingRequestPath, "w")
+    if len(sys.argv) < 3:
+        print("Unrecognized arguments: %s" % sys.argv)
+        sys.exit(1)
+    allow_new_request = validate_sb_request()
+    if allow_new_request:
+        with open(onGoingRequestPath, 'w') as f:
+            f.write(str(os.getpid()))
         os.makedirs(sspl_bundle_tmp)
         try:
             SSPLBundleObj = SSPLBundleGenerate()
-            SSPLBundleObj.generate_sspl_support_bundle(sys.argv, tmpFile)
+            SSPLBundleObj.generate_sspl_support_bundle(sys.argv)
+            sys.exit(0)
```

Comment:
Also return 1 in case of error 
```
    if allow_new_request:
        ...
    else:        
        sys.err.write("SSPL Support Bundle creation in progress. Existing...")
        sys.exit(1)
```